tests/backup/backup_basic_test.go

```diff
@@ -157,8 +157,8 @@ var _ = BeforeSuite(func() {
 			globalAWSBucketName = fmt.Sprintf("%s-%s", globalAWSBucketPrefix, bucketNameSuffix)
 			CreateBucket(provider, globalAWSBucketName)
 			log.Infof("Bucket created with name - %s", globalAWSBucketName)
-			s3SseTypeEnv := os.Getenv("S3_SSE_TYPE")
-			if s3SseTypeEnv != "" {
+			s3EncryptionPolicy := os.Getenv("S3_ENCRYPTION_POLICY")
```

Comment:
Can you make changes to .ssh file as well as currently it won't be there? Is this planned in the next PR?

Reply:
It is already merged as part of the previous PR